packages/okta-auth-js/lib/crypto.js

```diff
@@ -51,5 +62,6 @@ function verifyToken(idToken, key) {
 }
 
 module.exports = {
+  getOIDCHash: getOIDCHash,
```

Comment:
I'm in favor of only capitalizing the first letter of an acronym in variable names (getOidcHash).  Otherwise you end up with monstrosities like `XMLHttpRequest` where one is capitalized and one is only leading capitalized.  

Not something I'll argue in depth over, but worth mentioning.